app/models/landmark.rb

```diff
@@ -7,9 +7,25 @@ class Landmark < ActiveRecord::Base
   validates :image, presence: true
   validates :description, presence: true
   validates :user, presence: true
-
+  validates :average_review, inclusion: { in: %w(0 1 2 3 4 5) }
 
   def self.search(search)
     where("name ILIKE ?", "%#{search}%")
   end
+
+  def average_rating
+    reviews = self.reviews
```

Comment:
Check out the `average` method [provided by ActiveRecord](http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html), you may decide it does what you want here or could help you do it with fewer lines of code.